colrv1/noflags.toml

```diff
@@ -0,0 +1,3421 @@
+
+family = "Noto Color Emoji"
+output_file = "NotoColorEmoji-noflags.ttf"
+color_format = "glyf_colr_1"
+clipbox_quantization = 32
+
+[axis.wght]
+name = "Weight"
+default = 400
+
+[master.regular]
+style_name = "Regular"
+srcs = [
```

Comment:
maybe we should have a script similar to the one we added in https://github.com/googlefonts/color-fonts/blob/main/generate_config.py that regenerates the toml configs' based on the svg sources present in the noto-emoji/svg directory (or taken from the noto-emoji's Makefile) so there's only one source of truth as regards the list of sources. Say someone adds new emojis later, we may forget to update these lists in the all/noflags.toml and the font will miss some glyphs.